automatic/nexus-repository/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,25 @@
+$ErrorActionPreference = 'Stop'
+$ServiceName = 'nexus'
+$InstallFolder = "$env:ProgramData\Nexus"
+
+if (Test-Path "$Installfolder\bin\nexus.exe") {
+    if (Get-Process $ServiceName -ErrorAction SilentlyContinue) {
+        Get-Process $ServiceName | Stop-Process -Force
+    }
+
+    if (Get-Service $ServiceName -ErrorAction SilentlyContinue){
+        Stop-Service $ServiceName
+    }
+
+    Start-Sleep -Seconds 5
+
+    $ProcessArgs = @{
+        exeToRun       = "$InstallFolder\bin\nexus.exe"
+        Ssatements     = "/uninstall $ServiceName"
+        validExitCodes = @(0)
+    }
+    Start-ChocolateyProcessAsAdmin @ProcessArgs
```

Comment:
Any reason not to use `Uninstall-ChocolateyPackage` instead of starting the process itself?

Reply:
As above, this isn't strictly an installer/uninstaller so it seemed valid to do it this way. We don't want to pass in chocolateyInstallArguments to this call, so I don't think that function adds anything to what we're doing here.